src/utils/RAML.js

```diff
@@ -184,6 +184,9 @@ export default class RAML {
                             if (methodClassFunction === 'fetch') {
                                 debug(`Special case, this is a POST or PATCH at {id}, which is an update()`);
                                 methodClassFunction = 'update';
+                            } else if (methodClassFunction === 'list' && method.method === 'post') {
```

Comment:
Oh OK I see. Can you change it to this?

```
case 'patch':
                        case 'post':
                            if (method.method === 'post') {
                              //Catch POST methods only
                              if (methodClassFunction === 'list') {
                                debug(`Special case, this is a POST on collection, which is a create()`);
                                methodClassFunction = 'create';
                              }
                            }

                            //POST or PATCH
                            if (methodClassFunction === 'fetch') {
                                debug(`Special case, this is a POST or PATCH at {id}, which is an update()`);
                                methodClassFunction = 'update';
                            } 
                            break;
```

I think that is a bit easier to read.